feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+
+Set the chain ID:
+
+```bash
+simd config chain-id demo
+```
+
+Set the [keyring backend](https://docs.cosmos.network/v0.42/run-node/keyring.html#the-test-backend):
+
+```bash
+simd config keyring-backend test
+```
+
+## Key Setup
+
+You'll have to create a few test keys for your users.
+
+Add a key for Alice, the validator:
+
+```bash
+simd keys add alice
+```
+
+Add a key for Bob, the grantee:
+
+```bash
+simd keys add bob
+```
+
+And lastly, add a key for Carol, the granter:
+
+```bash
+simd keys add carol
+```
+
+If you'd like to see an overview of your keys, use:
+
+```bash
+simd keys list
+```
+
+In order to avoid having to copy-paste their address all the time, you should export them to variables that you can keep using during this tutorial:
+
+```bash
+export ALICE_KEY=$(simd keys show alice -a)
+export BOB_KEY=$(simd keys show bob -a)
+export CAROL_KEY=$(simd keys show carol -a)
+```
+
+## Chain Setup
+
+The following commands set up a chain using the simulation application (`simapp`).
+
+Initialize the node:
+
+```bash
+simd init test --chain-id demo
+```
+
+Alice will be your validator. Add Alice and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account alice 5000000000stake --keyring-backend test
+```
+
+Add Bob and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account bob 2000kudos --keyring-backend test
+```
+
+Note that Bob only has `kudos` tokens and won't be able to pay for any fees that might be needed.
+
+Add Carol and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account carol 1000000stake --keyring-backend test
+```
+
+Generate a transaction to add Alice to the initial validator set:
+
+```bash
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+
+```bash
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+
+```bash
+simd start
+```
+
+## Grant Allowance
+
+Before Bob can send `kudos` to Carol, you'll need to set up an allowance for Bob, so that Carol will pay for any gas fees the transaction might incur. Note that Alice is the validator, so it would not make sense to make her the granter as well because a validator would receive all the fees that the granter paid for. In that case, you wouldn't have been able to see the difference in her balance after a transaction was completed.
+
+The `BasicAllowance` is a permission for a grantee to use up fees until the `spend_limit` or `expiration` is reached. Create an allowance with a spend limit of `100000stake` and no expiration date:
+
+```bash
+simd tx feegrant grant $CAROL_KEY $BOB_KEY --from carol --spend-limit 100000stake
+```
+
+View the allowance:
+
+```bash
+simd q feegrant grants $BOB_KEY
+```
+
+## Send Tokens
+
+First, let's check the balances of Alice, Bob and Carol, so that you can later confirm if your transaction was successful:
```

Comment:
```suggestion
First, let's check the balances of Alice, Bob, and Carol. Verifying the initial balance provides a baseline so that you can later confirm if your transaction was successful:
```

Reply:
we love the serial comma https://developers.google.com/style/commas?hl=en